src/components/SideBar.astro

```diff
@@ -117,10 +117,10 @@
   pages.forEach((elem) => {
     if (url.includes(elem.id)) {
       isHome = false;
-      elem.classList.add("bg-base-300");
+      elem.classList.add("active");
```

Comment:
The first time I thought about the sidebar states I also thought about the 'active' class, but it happens that in this case the buttons are bigger than the ones in daisyUI and with many themes the color of the active class is very strong and contrasts a lot on the page and gets a lot of attention compared to the body of the page. 

That's why I decided to use this one that highlights them but with a lighter accent.

![image](https://user-images.githubusercontent.com/53962116/226484875-d3402bc4-07cf-46c6-b1b6-ec30a054ccb8.png)

![image](https://user-images.githubusercontent.com/53962116/226484981-e89649aa-c3c5-4d25-a997-13e8b754e7c1.png)

So I think it's better if we leave it here with that accent and not with active. What do you think ?

Reply:
Hi dear @manuelernestog 

First of all, thank you for your great project! It helped me a lot to quickly set up my own portfolio website.

Secondly, I agree with what you said. After all, you know about UI design more than I do. However, since some people might prefer the primary colour on the active item, what do you say I put a comment on top of it and add it to the README file as well? So it's easier to find.

Reply:
Please check [this commit](https://github.com/manuelernestog/astro-modern-personal-website/pull/11/commits/4e04b714f04773cdbb41d893886ab1874713374c) for the changes regarding this conversation.